Source/NSStoryboardSegue.m

```diff
@@ -143,6 +171,56 @@ - (void) perform
           RETAIN(w);
         }
     }
+  else if ([_kind isEqualToString: @"popover"])
+    {
+      NSPopover *po = [[NSPopover alloc] init];
```

Comment:
Eher does this object get released?

Reply:
A popover is a window... so I am not certain where it should be released.

Reply:
If I release it right away, then it closes... if I autorelease it, then it releases in the application autorelease loop once the method exits.   So I am not sure either.

Reply:
I have opened a PR for handling all popover related issues.  bug #71

Reply:
No, if it is a window then it will not be removed while it is displayed. Autoreleasing after the show call should be safe. Did you really test this?